docker/run-mysql.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
-# The CloudSQL console simply states "MySQL 5.7" so we may not match the minor version number
 # The docker version installed on DSP Jenkins does not like the "mysql" docker image, so we use "mysql/mysql-server"
-#     instead. Re-evaluate this sometime after Jenkins is no longer in use.
+#     instead.
+#     TODO: Move to mysql:8.0.35 once we no longer depend on Jenkins.
 MYSQL_IMAGE=mysql/mysql-server
-MYSQL_VERSION=5.7
+MYSQL_VERSION=8.0.32
```

Comment:
since we need to get Rawls off Jenkins by the end of April, this should be a relatively temporary state.